config/locales/fr.yml

```diff
@@ -41,7 +41,7 @@ fr:
     formats:
       default: "%d/%m/%Y"
       short: "%e %b"
-      long: "%e %B %Y"
+      long: "%A %e %B %Y"
```

Comment:
Might be proper to create a dedicated `search` or `trip` format to keep common behaviour.